wgsl/index.bs

```diff
@@ -793,6 +793,20 @@ to define a [=uniform buffer=] or a [=storage buffer=]. See [[#memory-layouts]].
   </xmp>
 </div>
 
+### Composite Types ### {#composite-types}
+
+A type is <dfn noexport>composite</dfn> if it has internal structure
+expressible as a composition of other types.
+The internal non-overlapping parts of the composite type are called <dfn noexport>components</dfn>.
```

Comment:
The non-overlapping sounds odd to me. Is this a property of a composite or a precondition?